src/rich-text-editor.component.js

```diff
@@ -220,5 +232,6 @@ RichTextEditor.defaultProps = {
   save: noop,
   placeholder: '',
   placeholderColor: '#CFCFCF',
-  pasteFn: null
+  pasteFn: noopWithReturn,
+  sanitizeHTML: noopWithReturn,
```

Comment:
Default behavior is to return the html unsanitized unless a user provides a sanitizeHTML prop.